test/TesterInternal/project.json

```diff
@@ -29,11 +29,9 @@
     "TestInternalGrainInterfaces": { "target": "project" },
     "TestInternalGrains": { "target": "project" },
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
-    "Newtonsoft.Json": "7.0.1",
     "System.Threading.Tasks.Parallel": "4.0.1",
     "WindowsAzure.Storage": "7.0.0",
     "xunit": "2.2.0-beta2-build3300",
-    "xunit.runner.visualstudio": "2.1.0",
```

Comment:
This was only required for DNX.  Under the dotnet cli the dotnet-test-xunit package is what's needed